wbooks/src/app/screens/BookData/styles.ts

```diff
@@ -0,0 +1,43 @@
+import { ImageStyle, StyleSheet, TextStyle, ViewStyle } from 'react-native';
+import { COLORS } from '@constants/colors';
+
+interface Styles {
+  container: ViewStyle;
+  bookCard: ViewStyle;
+  dataContainer: ViewStyle;
+  imageBook: ImageStyle;
+  title: TextStyle;
+  author: TextStyle;
+}
+
+export default StyleSheet.create<Styles>({
+  container: {
+    flex: 1,
+    marginTop: 35
+  },
+  bookCard: {
+    backgroundColor: COLORS.white,
+    borderRadius: 10,
+    padding: 30,
+    marginBottom: 10,
```

Comment:
use padding on the parent, if you were looking to separate parent and child.

Reply:
Done!